app/src/main/java/com/duckduckgo/app/global/MultiProcessApplication.kt

```diff
@@ -24,7 +24,7 @@ import android.os.Process
 
 abstract class MultiProcessApplication : Application() {
     private val shortProcessName: String by lazy {
-        currentProcessName?.substring(packageName.length) ?: "UNKNOWN"
+        currentProcessName?.substringAfter(delimiter = packageName, missingDelimiterValue = "UNKNOWN") ?: "UNKNOWN"
```

Comment:
The initial code was assuming the process name contains the package name. On some devices on OS versions below 9 (Android N) this comes as empty.